### DISCUSSION TITLE
Problema passaggio alunni classi successive

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#271] [Idee e proposte]
Buongiorno , come mai che nel progetto Giuaschool , al contrario del progetto LAMPSchool non è presente il passaggio automatico degli alunni nelle classi successive annualmente?Lo chiedo perché avendo usato per diversi anni LAMPSCHOOL c'era la possibilità nell'area amministratore di effettuare il passaggio automatico degli alunni  nelle classi successive , non essendoci questa possibilità in GIUASCHOOL, diventa tutto molto più difficile , soprattutto dovendo spostare alunno per alunno

**Comment by trinko** [comment]
Il codice dell'applicazione giua@school non è una copia di LAMPSchool, ma si ispira a quello. Ogni procedura è stata riscritta da zero e si è data priorità a quelle essenziali.

In altri termini, per me è più costoso, in termini di tempo, sviluppare la procedura per il passaggio di anno che importare a inizio anno di nuovo tutti gli alunni. Poi se qualcuno volesse contribuire a scrivere la procedura, ovviamente sarebbe una cosa utile.

Comunque la soluzione non è spostare alunno per alunno ma, come dicevo, importare nuovamente gli alunni con le classi modificate.
